src/model/TaskState.model.js

```diff
@@ -29,9 +29,7 @@ class TaskState extends Enumify {
   static RUNNING = new TaskState('running')
   static SUBMITTED = new TaskState('submitted')
   static PREPARING = new TaskState('preparing')
-  static QUEUED = new TaskState('queued')
   static WAITING = new TaskState('waiting')
-  static HELD = new TaskState('held')
```

Comment:
Ah, so the held state is gone too? I think some tests will need to be re-written or removed (the ones that are using `=== 'held'`).

Reply:
Yes, there's now a "held" indicator that gets attached to the actual task state.
As a temporary measure I replaced the value in the tests with the string `'held'`.